sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -414,7 +416,8 @@ ProgramManager::getOrCreateKernel(OSModuleHandle M, const context &Context,
                                   const program_impl *Prg) {
   if (DbgProgMgr > 0) {
     std::cerr << ">>> ProgramManager::getOrCreateKernel(" << M << ", "
-              << getRawSyclObjImpl(Context) << ", " << KernelName << ")\n";
+              << getRawSyclObjImpl(Context) << ", " << KernelName << ", "
+              << Prg->get() << ")\n";
```

Comment:
Is this change needed in this patch?
I suggest moving it to distinct patch.

Reply:
Ok, will add this enhancement for debugging ProgramManager as separate PR.